app/models/user.rb

```diff
@@ -67,6 +67,15 @@ def rating_record
     client.find_rating_info(participant_id).body.fetch('ratingRecord', {})
   end
 
+  # This is a helper method for pulling mhv_correlation_id
+  def mhv_correlation_id
+    @mhv_correlation_id ||= begin
+      return nil unless mvi.present?
+      return nil unless mvi[:mhv_id]
```

Comment:
Using return here will cause it to never set the instance variable, it returns all the way out of the begin/end block.